www/templates/control/main-control.html

```diff
@@ -118,9 +118,15 @@
         <div ng-click="checkConsent()" class="control-icon-button"><i class="ion-checkmark-circled"></i></div>
       </div>
       <div class="control-list-item">
-        <div class="control-list-text" ng-i18next>Dummy Notification in 5 Seconds</div>
+        <div class="control-list-text" ng-i18next>{{'control.dummy-notification'}}</div>
```

Comment:
Ah I see that this has been added in
https://github.com/e-mission/e-mission-translate/pull/30/files